salt/states/boto_sns.py

```diff
@@ -254,7 +254,9 @@ def absent(
             return ret
 
         for subscription in subscriptions:
-            unsubscribed = __salt__['boto_sns.unsubscribe'](name, subscription['SubscriptionArn'])
+            unsubscribed = __salt__['boto_sns.unsubscribe'](
+                name, subscription['SubscriptionArn'], region, key, keyid, profile
```

Comment:
Thanks for fixing this. Would you mind updating this to pass the positional kwargs in explicitly here? `region=region, key=key`, etc. We've had problems in the past with the boto modules where args get added down the line and the various function calls in the states might get missed in the update.